types/index.d.ts

```diff
@@ -0,0 +1,90 @@
+// Type definitions for Chalk
+// Definitions by: Thomas Sauer <https://github.com/t-sauer>
+
+export const enum Level {
+	None = 0,
+	Basic = 1,
+	Extended = 2,
+	TrueColor = 3
+}
+
+export interface ChalkOptions {
+	enabled?: boolean;
+	level?: Level;
+}
+
+export interface Chalk {
+	new (options: ChalkOptions): Chalk
+	(options: ChalkOptions): Chalk
+	(...text: string[]): string
+	(text: TemplateStringsArray, ...placeholders: string[]): string
+	constructor: Chalk
+	enabled: boolean
+	level: Level
+	supportsColor: {
+		level: Level,
+		hasBasic: boolean,
+		has256: boolean,
+		has16m: boolean
+	}
+	rgb: (r: number, g: number, b: number) => Chalk
```

Comment:
You could write them like `rgb(r: number, g: number, b: number): Chalk`. I believe this is the preferred way.